evrSupport/src/bsacompressRecord.c

```diff
@@ -69,15 +70,15 @@ rset bsacompressRSET={
 	RSETNUMBER,
 	report,
 	initialize,
-	init_record,
-	process,
+	(long (*)()) init_record,
```

Comment:
I suppose to be completely accurate, they should *all* be recast... but most of them are defined to be NULL.  Only the *real* routines require a cast to make the compile work.

Reply:
(These are required because of the addition of USE_TYPED_RSET.  The alternative is to let the compilation bitch about things being deprecated... and I don't like compilation warnings.)